base/src/com/google/idea/blaze/base/run/smrunner/BlazeXmlSchema.java

```diff
@@ -145,10 +145,10 @@ static TestSuite mergeSuites(List<TestSuite> suites) {
     @XmlAttribute public String result;
     @XmlAttribute public String time;
 
-    @XmlAttribute(name = "system-out")
```

Comment:
Thanks for the fix! Please also add tests for std-out and std-err to BlazeXmlSchemaTest to avoid future regressions.